Evo_Game_Function

```diff
@@ -0,0 +1,316 @@
+      Population <- vanishedList$Population
+      nAgent <- vanishedList$nAgent
+      agentField <- vanishedList$agentField
+      died <- vanishedList$died
+    
+      Entire_Individual <- rbind(Entire_Individual,Individual)
+      Individual <- Individual[Individual["fitness"]!= "vanishing",]
+      
+    }
+    
+    #Finished interval t, next time point
+    
+    t <- t + 1
+    
+    if(dim(Individual)[1] > 0) { # again, to not cause errors if no agent is still alive.
+      
+      priorIndividual <- Individual
+      Individual["t"] <- t
+      
+      #New born agents are now declared.
+      bornList <- born_Agents(Individual)
+      Individual <- bornList$Individual 
+      Population <- bornList$Population
+      nAgent <- bornList$nAgent
+      born <- bornList$born
+      
+      #Movement takes place of the other agents
+      Individual <- moving_Agents(Individual)
+      
+      #Depending on the order of birth, similar occupation of spaces in agents is resolved and the lower agent has to move
+      Individual <- distribute_Overlaying_Agents(Individual)
+      
+      #Computing the plot again
+      agentField <- field
+      agentField[Individual[["ind"]]] <- as.integer(row.names(Individual))
+      imgDat <- gen_Img_Dat(foodField,agentField,field)
+      imgPlot <- gen_Img_Plot(imgDat)
+      
+      # Output structure for the current time point is created (accounting for possible food or agent depletion)
+      metaData <- meta_Data()
+      if(max(!is.na(foodField)) > 0) {
+        foodCoord <- as.data.frame(cbind(which(!is.na(foodField), arr.ind = TRUE),
+                                       ind = which(!is.na(foodField))))
+      } else {
+        foodCoord <- NA
+      }
+      if(max(!is.na(agentField)) > 0) {
+        agentCoord <- Individual[c("ID","row","col","ind")]
+      } else {
+        agentCoord <- NA
+      }
+      Time_Series[[paste0("t",t)]] <- list(Image_Plot = imgPlot, Data_Plot = imgDat, Agent_Coordinates = agentCoord, Food_Coordinates = foodCoord, Meta_Data = metaData)
+    } else {
+      Time_Series[[paste0("t",t)]] <- NA 
+    }
+  }
+  
+  # Last dataframe update of the list of individuals that did not get the chance for sensing, action, and energy update
+  
+  if(dim(Individual)[1] > 0) {
+    Entire_Individual <- rbind(Entire_Individual,Individual)
+  }
+  
+  # Return of a list containing a list of spatial data per time point, and dataframes for the entire population and Individual
+  # status at a given time point
+  
+  return(list(Time_Series, Population, Entire_Individual))
+}
+
+
+
+
+# function space that needs to be loaded -------------------------------------------------------------------------------------
+
+library(tidyverse)
+library(ggplot2)
+library(reshape2)
+
+gen_Field_Pos <- function (field, n) {
+  FieldPos <- sample(1:length(field), size = n)
+  field[FieldPos] <- TRUE
+  return(field)
+}
+
+gen_Img_Dat <- function(foodField, agentField, field) {
```

Comment:
I get that it's convenient to use short object names, but this makes it really hard to read for a person who's not familiar with your code. You might want to chose more informative names.